Sources/AmplifyMapLibreAdapter/AmplifyMapLibre.swift

```diff
@@ -25,6 +27,19 @@ public class AmplifyMapLibre {
             }
         }
     }
+    
+    /// Creates an instance of MGLMapView configured to work with Amplify and Amazon
+    /// Location Service using the default map
+    public class func createMap() async throws -> MGLMapView {
+        AWSMapURLProtocol.register(sessionConfig: MGLNetworkConfiguration.sharedManager.sessionConfiguration)
+        do {
+            let map = try await Amplify.Geo.defaultMap()
+            return await MGLMapView(frame: .zero, styleURL: map.styleURL)
+        } catch {
+            throw error as! Geo.Error
+        }
+        
+    }
```

Comment:
```suggestion
    @MainActor public class func createMap() async throws -> MGLMapView {
        AWSMapURLProtocol.register(sessionConfig: MGLNetworkConfiguration.sharedManager.sessionConfiguration)
        let map = try await Amplify.Geo.defaultMap()
        return MGLMapView(frame: .zero, styleURL: map.styleURL)
    }
```

Reply:
There's no need to do the `do catch` dance here. Also let's avoid force casting, even if we "know" we're only going to be throwing `Geo.Error`. It doesn't provide any value to the caller and could end up being problematic in the future if we ever throw a different error without updating this.

Also, `UIView` inits are `@MainActor` isolated, that's why you needed to `await` on the `MGLMapView.init` despite it not actually being an `async` initializer. Since this method will most likely be called from some UI context, I believe annotating the method as `@MainActor` is the more "correct" way. I'm ok with dropping `@MainActor` and sticking with `await MGLMapView(...)` if there's a good reason to though.